### PR TITLE
[rfc] Style

### PR DESCRIPTION
**Pull request by jongold**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+// define some abstract style transformations
+// bumpFontSize :: CSS -> CSS
+const bumpFontSize = evolve({
+  fontSize: add(4),
+});
+
+// darkenText :: CSS -> CSS
+const darkenText = evolve({
+  color: c => chroma(a).darken(),
```
Comment (justingreenberg) [inline README.md]: awesome work!! 👍  the param here may be typo?

```diff
- color: c => chroma(a).darken(),
+ color: a => chroma(a).darken(),
```

...assuming a is for alpha and you wouldn't create namespace conflict with compose alias :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+Style.of({ color: 'red' }).equals(Style.of({ color: 'red' }))
+// true
+
+Style.of({ color: 'red' }).equals(Style.of({ color: 'blue' }))
+// false
+
+```
+
+#### concat
+##### `#concat :: Style a ~> a ~> Style a`
```
Comment (buzzdecafe) [inline README.md]: this looks more like `append` to me? Doesn't `concat` need a semigroup for an argument? e.g.

`[1, 2, 3].concat([4, 5, 6])`

File: README.md (hunk 1 above)
Comment (jongold) [inline README.md]: this is definitely a typo, thanks

File: README.md (hunk 2 above)
Comment (jongold) [inline README.md]: you're correct, thanks :)

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+// @flow
+
+// Style :: => (Props -> CSS) -> Style CSS
+function Style(f) {
+  if (!(this instanceof Style)) {
+    return new Style(f);
+  }
+
+  this.__value = typeof f === "function" ? f : () => f;
```
Comment (buzzdecafe) [inline src/index.js]: i'm not a fan of this defaulting. IMO either always take a function or change the type sig to describe what legal arguments are (namely anything)

**Pull request merged**